app/components/list-item/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ListItemType } from '@/app/app-types'
+
+type ListItemProps = {
```

Comment:
Could you please create this as interface instead of type

Reply:
Updated as interface.